runtime/src/js/misc.js

```diff
@@ -26,6 +26,10 @@ const createDir = (dirName) => {
   }
 };
 
+export const insideFlatpak = () => {
```

Comment:
Remove the **export** keyword from here, and add 
`insideFlatpak: insideFlatpak,` into the module.exports = {...} in the same file.

Reply:
Changed as suggested